src/Event/Productions/ProductionCommandHandler.php

```diff
@@ -31,6 +41,11 @@ public function handleGroupEventsAsProduction(GroupEventsAsProduction $command):
             $command->getName(),
             $command->getEventIds()
         );
+
+        foreach ($command->getEventIds() as $eventId) {
```

Comment:
Maybe useful to bundle multiple exceptions when multiple events don't exist?

Reply:
The UI should only give us valid events. If there are non-existing events, it's mostly because someone tampered with the request. In those situations, I don't care too much about optimising error handling. Do you think this is a must-have?

Reply:
Is it part of the API?

Reply:
Yes, but the API is only privately consumed by an internal UI used by a handful of people